_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -346,7 +346,7 @@ newsletter's author.
 [cjp rz]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001498.html
 [zmn rz]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001547.html
 [zmn rz packetswitch]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001553.html
-[cl plugin]: https://github.com/ElementsProject/lightning/blob/master/doc/plugins.md
+[cl plugin]: https://github.com/ElementsProject/lightning/blob/master/doc/PLUGINS.md
```

Comment:
Have you considered linking to a specific commits, eg https://github.com/ElementsProject/lightning/blob/018a3f1d58e38e464e32cbee26826a0bd63aab59/doc/PLUGINS.md to prevent links to files in repos from breaking like this?

Reply:
Yeah.  I think it's a toss-up between ensuring our links don't break and helping users find up-to-date information.  For now, I opt towards the later, but I have no strong preference.  FYI, the htmlproofer command used in the Makefile to check internal links (among other things) can also check external links---this is noisy and wasteful of network resources, so it shouldn't be done automatically from Travis, but we can run it every few months and fix up links then.

Reply:
I didn't know Travis now supported scheduled jobs, nice!  I've missed that feature from Jenkins.  I'll open a PR implementing that for our Makefile, and if it works for you, you should have access to add the scheduled job.